tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -129,17 +130,25 @@ fi
 
 ensure_yq
 
-info "Running tests with bats version: $(bats --version)"
+report_dir="${kubernetes_dir}/reports/$(date +'%F-%T')"
+mkdir -p "$report_dir"
```

Comment:
shellcheck is gonna want this in braces. Same for a few other new variables here

Reply:
ah, ok, got it. Thanks @stevenhorsman !